.github/workflows/build_and_test.yml

```diff
@@ -841,7 +841,8 @@ jobs:
           ./vm-builder \
             -spec=compute/vm-image-spec-${{ matrix.version.debian }}.yaml \
             -src=neondatabase/compute-node-${{ matrix.version.pg }}:${{ needs.tag.outputs.build-tag }} \
-            -dst=neondatabase/vm-compute-node-${{ matrix.version.pg }}:${{ needs.tag.outputs.build-tag }}
+            -dst=neondatabase/vm-compute-node-${{ matrix.version.pg }}:${{ needs.tag.outputs.build-tag }} \
+            -size=1200M
```

Comment:
Let's copy changes from https://github.com/neondatabase/neon/pull/9498 to avoid merge conflicts (I am talking about `-size=2G \ `)

https://github.com/neondatabase/neon/blob/8b4a1353b9dedb4e1687de4aca8f01284944c45c/.github/workflows/build_and_test.yml#L839-L845